docusaurus/docs/user-docs/releases/managing-a-release.md

```diff
@@ -35,12 +36,13 @@ You can rename a release. To do so, while on a release page:
 3. In the modal, change the name of the release in the _Name_ field.
 4. Click **Continue** to save the change.
 
-<!-- TODO: re-add when implemented -->
-<!-- ## Choose how entries are grouped
+## Choose how entries are grouped
 
-A release page can display entries either grouped by locales or by content-type. To change how entries are grouped, click the **Group by …** dropdown and select an option from the list. -->
+:::callout 🚧 Beta feature
+Grouping is only available in Strapi v4.18.0-beta.
```

Comment:
I don't think we need to specify the grouping here. We can just say use the beta release for the most up to date version of Releases.

```suggestion
Use the following command to install the latest version of this feature:
npx create-strapi-app@beta
```